backend/src/v5/utils/permissions/permissions.constants.js

```diff
@@ -0,0 +1,22 @@
+/**
+ *  Copyright (C) 2021 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+const PermConst = {};
+
+PermConst.teamspaceAdmin = 'teamspace_admin';
```

Comment:
***Just a suggestion:**
change it to `PERM` or `PERMISSIONS`  or `PERMISSION_CONST` with the constant casing?

I dont know why but this looks a bit more like constants definition, the other way seems more like an object with behaviour. 

```suggestion
const PERMISSIONS_CONST = {
	teamspaceAdmin: 'teamspace_admin',
};
```

Reply:
as discussed, changing `teamspaceAdmin` to `TEAMSPACE_ADMIN` as that's the variable that gets exported.